codecs/opus/src/test/java/org/restcomm/media/codec/opus/OpusCodecTest.java

```diff
@@ -71,9 +79,36 @@ public void tearDown() throws Exception {
      */
     @Test
     public void testCodec() {
-    	OpusJni opus = new OpusJni();
-    	opus.setOpusObserverNative(this);
-    	opus.sayHelloNative();
+    	
+    	try {
+	        FileInputStream inputStream = new FileInputStream("src\\main\\jni\\test_sound_mono_48.pcm");
+	        FileOutputStream outputStream = new FileOutputStream("src\\main\\jni\\test_sound_mono_48_decoded.pcm", false);
```

Comment:
Create temporary file instead and make sure we delete it after the test.

Example: `Path tempFile = Files.createTempFile("tempfiles", ".tmp");`

Reply:
Fixed